### PR TITLE
chore(docs): update README and quick.md files 

### PR DESCRIPTION
**Pull request by prateekpandey14**
Signed-off-by: prateekpandey14 <prateek.pandey@mayadata.io>

### CODE REVIEW

File: docs/tutorial/cspc/tuning/tune.md [hunk 1]
```diff
@@ -26,19 +26,19 @@ metadata:
 spec:
   resources:
     requests:
-      memory: "64Mi"
+      memory: "2Gi"
       cpu: "250m"
     limits:
-      memory: "128Mi"
+      memory: "4Gi"
       cpu: "500m"
```
Comment (mittachaitu) [inline docs/tutorial/cspc/tuning/tune.md:33]: Not sure do we need to specify the show CPU limits?

File: README.md [hunk 2]
```diff
@@ -63,6 +64,12 @@ perform pool operations e.g pool provisioning.
 **Note:** It is not recommended to modify the CSPI CR and pool-manager in the running cluster unless you know what you are 
 trying to do. CSPC should be the only point of interaction.
 
+Once the CStor pool get provisioned successfully, admin/user can create PVC to provision csi CStor volumes, when a user
```
Comment (sonasingh46) [inline README.md:67]: n_p:
From control plane perspective and hence from SRE and users -- CStor Pool is not provisioned.
`CStorPoolCluster` is provisioned. CStor pools are the end result of CSPC provisioning. 
So I would prefer : 

`Once the CStorPoolCluster get provisioned`

Comment (sonasingh46) [reply README.md:67]: If we are mentioning like this at other places and my idea sounds good then we will fix it at all other places.

Comment (sonasingh46) [reply README.md:67]: n_p: 
`provision csi CStor volumes,` ---> `provision csi CStor volumes.`
Notice the comma changed to a period.

File: README.md [hunk 3]
```diff
@@ -63,6 +64,12 @@ perform pool operations e.g pool provisioning.
...
+creates PVC, CStor CSI driver creates CStorVolumeConfig(CVC) resource, managed and reconcile by the **cvc-controller** which creates
+different volume specific resources for each persistent volume, later managed by their respective controllers, more info
```
Comment (sonasingh46) [inline README.md:69]: There is only one persistent volume for a PVC. So I think `each` is not required.

Comment (sonasingh46) [reply README.md:69]: We can delete this part `for each persistent volume`

File: docs/quick.md [hunk 4]
```diff
@@ -19,7 +20,15 @@ meets the following prerequisites:
...
 
-### Insallation
+Installation:
+-------------
+
+Note:
+------
+
+It's always a better idea to install latest tag [release](https://github.com/openebs/cstor-operators/releases) yamls, unless we build the images locally. The latest release specific yamls of cstor-operators,
+csi-operators and ndm-operators, can be fetched and apply directly from [release pages](https://github.com/openebs/charts/tree/gh-pages).
```
Comment (sonasingh46) [inline docs/quick.md:30]: n_p:
`apply` --->`applied`

File: docs/quick.md [hunk 5]
```diff
@@ -19,7 +20,15 @@ meets the following prerequisites:
...
+Since below mentioned resources are already created as part of release yamls, we can skip 1-7 steps and continue from step 8 onwards.
```
Comment (sonasingh46) [inline docs/quick.md:31]: n_p:
`below` --> `the below`

Comment (sonasingh46) [reply docs/quick.md:31]: `release` --> `release`

File: README.md (hunk 3 above)
Comment (prateekpandey14) [reply README.md:69]: Here we are talking in reference to cvc-operator, which looks ok to me

**Pull request merged**